### PR TITLE
ER-287: Added form alter and loan text

### PR DESCRIPTION
**Pull request by martinyde**
Added form alter to user_admin_settings_form
Added panels_pane_content_alter to user loan pane

### CODE REVIEW

File: sites/all/modules/reol_loan/reol_loan.module [hunk 1]
```diff
@@ -265,3 +265,41 @@ function reol_loan_delete_reservations_ajax($user, $local_ids) {
...
+ * Implements hook_panels_pane_content_alter().
+ *
+ * Change the content of the loan panel pane on user pages.
+ */
+function reol_loan_panels_pane_content_alter($content, $pane, $args, $contexts) {
+  if ($pane->type =='loans') {
+    if ($pane->subtype == 'loans') {
+      if (!empty($content->content)) {
+        // Add text to top of loan block.
+        $content->content = '<div>' . variable_get('ereol_loan_text', '')['value'] . '</div>' . $content->content;
```
Comment (cableman) [inline sites/all/modules/reol_loan/reol_loan.module]: `variable_get('ereol_loan_text', '')['value']`will fail if the variable is not set.

Could be changed to `variable_get('ereol_loan_text', array('value' => ''))['value']`

File: sites/all/modules/reol_loan/reol_loan.module [hunk 2]
```diff
@@ -265,3 +265,41 @@ function reol_loan_delete_reservations_ajax($user, $local_ids) {
...
+    '#title' => t('Ereol user settings'),
+    '#weight' => 0,
+    '#tree' => FALSE,
+  );
+
+  $form['ereol_user']['ereol_loan_text'] = array(
+    '#type' => 'text_format',
+    '#title' => t('Loan text'),
+    '#format' => 'ding_wysiwyg',
+    '#default_value' => variable_get('ereol_loan_text', '')['value'],
```
Comment (cableman) [inline sites/all/modules/reol_loan/reol_loan.module]: variable_get('ereol_loan_text', '')['value']will fail if the variable is not set.

Could be changed to variable_get('ereol_loan_text', array('value' => ''))['value']

Comment (martinyde) [reply sites/all/modules/reol_loan/reol_loan.module]: Modified variable get

File: sites/all/modules/reol_loan/reol_loan.module (hunk 1 above)
Comment (martinyde) [reply sites/all/modules/reol_loan/reol_loan.module]: Modified variable get

**Comment by cableman** [comment]
I have set the label "Ready for merge" as I don't known if this i under test any where? If not it's ready for merge into develop.

File: sites/all/modules/reol_loan/reol_loan.module [hunk 3]
```diff
@@ -265,3 +265,43 @@ function reol_loan_delete_reservations_ajax($user, $local_ids) {
...
+/**
+ * Implements hook_panels_pane_content_alter().
+ *
+ * Change the content of the loan panel pane on user pages.
+ */
+function reol_loan_panels_pane_content_alter($content, $pane, $args, $contexts) {
+  if ($pane->type =='loans') {
+    if ($pane->subtype == 'loans') {
+      if (!empty($content->content)) {
+        if (isset(variable_get('ereol_loan_text')['value'])) {
```
Comment (rimi-itk) [inline sites/all/modules/reol_loan/reol_loan.module]: Bør `isset` ikke kun bruges med variabler som argument?

Comment (cableman) [reply sites/all/modules/reol_loan/reol_loan.module]: Did think about that one... but I have seen this pattern in drupal 7 before.

**Pull request merged**